client/utils/fakeFilterFlags.js

```diff
@@ -0,0 +1,29 @@
+const fakeFilterFlags = (data) => {
+  // *******FAKE DATA GENERATOR************
+  data.slice(0, Math.floor(data.length / 3)).forEach((d) => {
+    const node = d;
+    node.wifi = true;
+    node.food = true;
+    node.open = false;
+  });
+  data.slice(Math.floor(data.length / 3), Math.floor((data.length * 2) / 3)).forEach((d) => {
+    const node = d;
+    node.wifi = false;
+    node.food = true;
+    node.open = true;
+  });
+  data.slice(Math.floor((data.length * 2) / 3)).forEach((d) => {
+    const node = d;
+    node.wifi = true;
+    node.food = false;
+    node.open = true;
+  });
+
+  data[0].wifi = true;
```

Comment:
Oh I'm glad you asked that, because I'm fixing the flags until a more legitimate method works. This is in order to have at least one parklet that has all three flags to display the behavior when all three filters are active.